index.bs

```diff
@@ -1130,6 +1166,314 @@ The following extended attributes are applicable to interfaces:
+
+A [=mixin=] is a specification of a set of <dfn export lt="mixin member">mixin members</dfn>
+(matching <emu-nt><a href="#prod-MixinMembers">MixinMembers</a></emu-nt>),
+which are the [=constants=], [=regular operations=], [=regular attributes=], and [=stringifiers=]
+that appear between the braces in the [=mixin=] [=declaration=].
+
+These [=constants=], [=regular operations=], [=regular attributes=], and [=stringifiers=]
+describe the behaviors that can be implemented by an object,
+as if they were specified on the [=interface=] that [=includes=] them.
+
+[=Static attributes=], [=static operations=], [=special operations=], except for [=stringifiers=], and
+[=iterable declaration|iterable=], [=maplike declaration|maplike=], and [=setlike declarations=]
+cannot appear in [=mixin=] [=declarations=].
+
+As with interfaces, the IDL for [=mixins=] can be split into multiple parts by using
+<dfn export>partial mixin</dfn> definitions
+(matching <emu-t>partial</emu-t> <emu-t>interface</emu-t> <emu-nt><a href="#prod-MixinRest">MixinRest</a></emu-nt>).
+The [=identifier=] of a [=partial mixin=] [=definition=] must
+be the same as the [=identifier=] of a [=mixin=] [=definition=].
+All of the [=mixin members|members=] that appear on each of the [=partial mixin=] [=definitions=]
+are considered to be [=members=] of the [=mixin=] itself,
+and—by extension—of the [=interfaces=] that [=include=] the [=mixin=].
+
+<pre class="webidl" class="syntax">
+    interface mixin <mark>SomeMixin</mark> {
+      /* mixin_members... */
+    };
+
+    partial interface mixin <mark>SomeMixin</mark> {
+      /* mixin_members... */
+    };
+</pre>
+
+The order that members appear in has significance for property enumeration
+in the <a href="#es-namespaces">ECMAScript binding</a>.
+
+Note that unlike [=interfaces=] or [=dictionaries=], [=mixins=] do not create types.
+
+Of the extended attributes defined in this specification,
+only the [{{Exposed}}] and [{{SecureContext}}] extended attributes are applicable to [=mixins=].
+
+An <dfn>includes statement</dfn> is a definition
+(matching <emu-nt><a href="#prod-IncludesStatement">IncludesStatement</a></emu-nt>)
+used to declare that all objects implementing an [=interface=] |I| (identified by the first [=identifier=])
+must additionally implement [=mixin=] |M| (identified by the second identifier).
+[=Interface=] |I| is said to <dfn>include</dfn> [=mixin=] |M|.
+
+<pre class="webidl" class="syntax">
+    interface_identifier includes mixin_indentifier;
+</pre>
+
+The first [=identifier=] must reference a [=callback interface|non-callback=] [=interface=] |I|.
+The second identifier must reference a [=mixin=] |M|.
+
+Each [=interface=] |I| that [=includes=] a [=mixin=] |M| must receive
+a copy of each of the [=mixin members|members=] of |M|.
+Each [=mixin member=] cooy is treated as if it had been declared on |I|.
+Its <dfn>host interface</dfn> is |I|.
```

Comment:
This paragraph is a bit weird as a "must" statement, but I get what it's trying to do. Maybe it could be made less weird by inserting something like "Per the appropriate language binding" or something, since as an implementer, the language-binding specific normative requirements are going to be the actually interesting ones?

Reply:
I reformulated that paragraph. LMK if it's now clearer.